changelog.d/20250422_185842_severine.bonnechere_handle_tar_gz_in_docker_images.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- Fixed error when scanning `.tar.gz` files inside docker images.
```

Comment:
nit: it's not really .tar.gz files inside docker images (as in a Dockerfile with a `COPY archive.tar.gz` command), it's more about tar.gz-compressed layers.